cleo_sdk/CLEO_Utils.h

```diff
@@ -418,6 +420,9 @@ namespace CLEO
     #define OPCODE_READ_PARAM_FLOAT() _readParamFloat(thread).fParam; \
         if (!IsLegacyScript(thread) && !_paramWasFloat()) { SHOW_ERROR("Input argument #%d expected to be float, got %s in script %s\nScript suspended.", CLEO_GetParamsHandledCount(), CLEO::ToKindStr(_lastParamType, _lastParamArrayType), CLEO::ScriptInfoStr(thread).c_str()); return thread->Suspend(); }
 
+    #define OPCODE_READ_PARAM_ANY() _readParam(thread).dwParam; \
+        if (!_paramWasInt() && !_paramWasFloat()) { SHOW_ERROR("Input argument #%d expected to be simple-type, got %s in script %s\nScript suspended.", CLEO_GetParamsHandledCount(), CLEO::ToKindStr(_lastParamType, _lastParamArrayType), CLEO::ScriptInfoStr(thread).c_str()); return thread->Suspend(); }
```

Comment:
OPCODE_READ_PARAM_ANY_NUMBER

Input argument #%d expected to be a number

Reply:
It is more about single 'variable cell' 32bit data than number actually. 
Calling OPCODE_READ_PARAM_NUMBER() returning DWORD will cause confusion when float is provided. User perhaps will expect to receive 16 when 16.0 was provided, meanwhile it will be just crude cast of float's data.

Reply:
OPCODE_READ_PARAM_DWORD?

Reply:
There are already
```
    OPCODE_READ_PARAM_INT()
    OPCODE_READ_PARAM_UINT()
```
and this will look like just another variation of integer.
Previously I also considered "VAR" but is suggest variable, or "ANY32"

Reply:
anything in memory is another variation of integer)) I think DWORD is very explicit and concrete

Reply:
DWORD is integer type. Arithmetic operation works as on integer type, assigning float to DWORD causes conversion.

Reply:
you literally use _readParam(thread).dwParam there

Reply:
Yes and returned type is DWORD. But as mentioned it is not good name for the macro itself.

Updated to OPCODE_WRITE_PARAM_ANY32, as it excludes strings.